### PR TITLE
Add initial flow control windows

### LINKED ISSUE #63: Initial flow control windows
The document currently does not specify what the initial flow control windows are, nor how to change those.

We have two options:
* Add WT_SETTINGS capsule
* Put those into headers

### PR DESCRIPTION
**Pull request by vasilvv**
Fixes #63

### CODE REVIEW

File: draft-ietf-webtrans-http2.md [hunk 1]
```diff
@@ -784,6 +784,29 @@ WebTransport Data
                                     WebTransport Data
 ~~~
 
+# WebTransport Header Fields
+
+WebTransport over HTTP/2 uses HTTP header fields to communicate the initial
+values of the flow control windows, similar to how QUIC uses transport
+parameters.  All of the fields defined in this section are Item Structured
+Fields (Section 3.3 of {{!RFC8941}}); all of the field values MUST be Integer.
```
Comment (martinthomson) [inline draft-ietf-webtrans-http2.md:792]:
```suggestion
Fields ({{Section 3.3 of !RFC8941}}); all of the field values MUST be Integer.
```

**Review by martinthomson — COMMENTED** [review]
The header field names here are pretty scary.  The prevailing attitude seems to be that header fields are cheap, but when I see a 48 character (!) name, that crosses a line for me.

I'll take comments about settings vs. fields to the issue.

**Review by martinthomson — COMMENTED** [review]
Any reason why you did not do a single header field, like:

```
WebTransport-Init: bl=100, br=200, u=300, s=10
```

Then at least the common repetitive stuff isn't repeated.

File: draft-ietf-webtrans-http2.md [hunk 2]
```diff
@@ -784,6 +784,29 @@ WebTransport Data
...
+If any of the fields cannot be parsed correctly or does not have a correct
+type, the peer MUST reset the CONNECT stream.
```
Comment (ekinnear) [inline draft-ietf-webtrans-http2.md:794]: Do we reset or reject with some error code/response?

**Review by ekinnear — APPROVED** [review]

File: draft-ietf-webtrans-http2.md (hunk 2 above)
Comment (DavidSchinazi) [reply draft-ietf-webtrans-http2.md:794]: Usual way to handle this is to say ", the peer MUST treat the message as malformed' then the version-specific rules kick in

**Comment by DavidSchinazi** [comment]
Chair: discussed during editors' call, plan is to go with Martin's simpler proposed encoding. We might want to discuss non-zero defaults later, but that'll be its own separate issue.

**Review by DavidSchinazi — APPROVED** [review]

**Review by afrind — APPROVED** [review]

**Pull request merged**